routes/farmer/farmer.js

```diff
@@ -258,23 +259,44 @@ exports.editCommodity = function(req, res) {
  * @param res
  */
 exports.updateFarmById = function(req, res) {
-    if(common.isAuthenticated(req, res)) {
-        model.Farm.update({_id: req.params.farm_id}, req.body, function (err, response) {
-            if (err) {
-                handleDBError(err, res);
-            } else {
-                //check if any document got modified
-                if (response.nModified != 0) {
-                    res.send(response);
+
```

Comment:
@nick22891 checking if the user is authenticated must be done on all endpoints. Where this is not done, a reason must be given.